e2e/setup/scenarios.screenshot.ts

```diff
@@ -1,27 +1,42 @@
 import {cucumber} from '../transformer/gherkin';
-import type {TestEnv} from './scenarious.world';
+
+import type {TestEnv} from './scenarios.env';
 import type {MatchImageSnapshotOptions} from 'jest-image-snapshot';
 
 const DIFF_CONFIG: MatchImageSnapshotOptions = {
   customDiffDir: '.diff'
 };
 
-cucumber.defineRule('take a screenshot', async (world: TestEnv) => {
-  world.screenshots.push(await page.screenshot());
+cucumber.defineRule('take a screenshot', async ({screenshots}: TestEnv) => {
+  screenshots.push(await page.screenshot());
+});
+
+cucumber.defineRule('take a screenshot of a full page', async ({screenshots}: TestEnv) => {
+  screenshots.push(await page.screenshot({fullPage: true}));
```

Comment:
Can we save screenshots in WEBP format?

Reply:
> Can we save screenshots in WEBP format?

Not yet, but there's a ticket opened for it https://github.com/exadel-inc/esl/issues/2393